libs/autofile/autofile.go

```diff
@@ -114,10 +125,18 @@ func (af *AutoFile) Sync() error {
 }
 
 func (af *AutoFile) openFile() error {
-	file, err := os.OpenFile(af.Path, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0600)
+	file, err := os.OpenFile(af.Path, os.O_RDWR|os.O_CREATE|os.O_APPEND, autoFilePerms)
 	if err != nil {
 		return err
 	}
+	fileInfo, err := file.Stat()
+	if err != nil {
+		return err
+	}
+	if fileInfo.Mode() != autoFilePerms {
+		return PermissionsChangedErr(fmt.Sprintf("file: [%v]\nexpected file permissions: %v, got: %v",
```

Comment:
Additionally this way of formatting, I know brought it up before, is confusing. There is no clear rule to when to break a line and recognising where one construct starts and another requires a lot of effort. More consistently it should look like this:

``` go
	return PermissionsChangedErr(
		fmt.Sprintf(
			"file: [%v]\nexpected file permissions: %v, got: %v",
			file.Name(),
			autoFilePerms,
			fileInfo.Mode(),
		),
	)
```

This is allows for faster reading and faster editing of the code.